index.html

```diff
@@ -11,167 +11,172 @@
     <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
   </head>
 
-<body>
-  <!--HEADER, BUILT BY CAMPER-->
+  <body>
+    <!--HEADER, BUILT BY CAMPER-->
     <header>
       <div class="header-media">
+        <!--video-->
+        <video autoplay muted loop>
+        </video>
+
         <!--secondary header-->
         <div class="secondary-header">
-          <p>Nonprofit Website Design Agency</p>
-          <p>Questions? Call us at 800-475-4590</p>
+          <h1>Nonprofit Website Design Agency</h1>
+          <h2>Questions? Call us at 800-475-4590</h2>
         </div>
 
-        <!--main header-->
-        <div class="main-header">
-          <h1 class="company-name">Eleva<span>tion</span></h1>
-          <a href="#">Contact Us</a>
-          <a href="#">1For1 Match</a>
-          <span class="fa fa-search"></span>
-          <a href="#">Menu <span class="fa fa-bars"></span></a>
+        <!--primary header-->
+        <div class="primary-header">
+          <div class="logo-content">
+            <img class="logo" src="./resources/images/logo.png" alt="ElevationWeb">
+          </div>
+
+          <div class="button-content">
+            <a class="button contact-us" href="#">Contact Us</a>
+            <a class="button one-match" href="#">1For1 Match</a>
+            <span class="fa fa-search search"></span>
+            <a class="menu" href="#">Menu</a>
+            <a class="menu-bars" href="#"><span class="fa fa-bars fa-2x"></span></a>
+          </div>
         </div>
 
-     <!-- BANNER, BUILT BY -->
+        <!--BANNER, BUILT BY-->
         <div class="banner">
         </div>
-     </div>
+      </div>
+
+      <!--CLIENTS, BUILT BY CAMPER-->
+      <div class="clients">
+      </div>
     </header>
 
 
-<!--BODY, BUILT BY CMOORE-->
```

Comment:
The reason this section looks like it was changed is because the spacing that my text editor uses is different from the current code. You can see this better in the split view (https://github.com/chingu-voyage3/toucans-06/pull/46/files?diff=split). Generally, my editor uses two spaces instead of indents.